testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/openshift/OpenShiftTokenReviewEndpointTest.java

```diff
@@ -228,29 +241,10 @@ public void customScopes() {
     }
 
     @Test
-    public void emptyScope() throws Exception {
-        ClientRepresentation clientRep = testRealm().clients().findByClientId("test-app").get(0);
-        List<ClientScopeRepresentation> scopesBefore = testRealm().clients().get(clientRep.getId()).getDefaultClientScopes();
-
-        try (ClientAttributeUpdater cau = ClientAttributeUpdater.forClient(adminClient, "test", clientRep.getClientId())
-                .setConsentRequired(false)
-                .setFullScopeAllowed(false)
-                .setDefaultClientScopes(Collections.EMPTY_LIST)
-                .update()) {
-
-            oauth.openid(false);
-            try {
-                new Review()
-                        .invoke()
-                        .assertSuccess()
-                        .assertEmptyScope();
-            } finally {
-                oauth.openid(true);
-            }
-        }
-        // The default client scopes should be same like before.
-        int scopesAfterSize = testRealm().clients().get(clientRep.getId()).getDefaultClientScopes().size();
-        assertEquals(scopesBefore.size(), scopesAfterSize);
+    public void emptyAudience() {
+        new Review().username("empty-audience")
+                .invoke()
```

Comment:
Is this really invoked without audience? I can see it added here:
https://github.com/keycloak/keycloak/blob/39b05b499c238899bb9d02ef681e500dc58fd38e/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/openshift/OpenShiftTokenReviewEndpointTest.java#L379
But maybe I'm just missing something. :)

Reply:
It's the other way around. The `account` audience is being added to the **expected** audience. The Token itself will have empty audience emitted from the Token Endpoint.